data/css/_project-setup.scss

```diff
@@ -6,6 +6,3 @@
 
 $images-path: 'resource:///com/endlessm/knowledge/data/images/';
```

Comment:
Aside from Matt's comments, we also have ruby-sass/compass bits in https://github.com/endlessm/eos-subscriptions-apps